src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/config/VisitSchedulerExceptionHandler.kt

```diff
@@ -74,7 +75,7 @@ class VisitSchedulerExceptionHandler {
 
   @ExceptionHandler(MissingServletRequestParameterException::class)
   fun handleValidationException(e: MissingServletRequestParameterException): ResponseEntity<ErrorResponse> {
-    log.debug("Bad Request (400) returned", e.message)
+    log.debug("Bad Request (400) returned {}", e.message)
```

Comment:
$e.message can we do that in Kotlin?

Reply:
Sure we would have to update all of the debug for this class, this change is just to put in the missing syntax.  We can create a ticket to review the debug for the whole app

Reply:
another time... no probs